src/server/db_slice.cc

```diff
@@ -467,7 +483,7 @@ tuple<PrimeIterator, ExpireIterator, bool> DbSlice::AddOrFind2(const Context& cn
 
     events_.garbage_collected = db.prime.garbage_collected();
     events_.stash_unloaded = db.prime.stash_unloaded();
-    events_.evicted_keys += evp.evicted();
```

Comment:
why it was deleted?

Reply:
no, it shouldn't be removed. I did this to temporarily test the counter increment from my code... need to put it back. fixed.